README.rst

```diff
@@ -3,16 +3,12 @@ ProjectKorra Official Wiki
 ==========================
 
 .. image:: https://readthedocs.org/projects/projectkorrawiki/badge/?version=latest
-    :target: https://projectkorrawiki.readthedocs.io/en/latest/?badge=latest
+    :target: https://wiki.projectkorra.com/en/latest/
```

Comment:
Should be `https://wiki.projectkorra.com/`.

Reply:
It is?

Reply:
No, you really shouldn't be adding the `/en/latest` to the end of the link URL.

Reply:
Oh thats what you meant, gotcha